_dashboards/visualize/maps.md

```diff
@@ -1,22 +1,35 @@
 ---
 layout: default
-title: Using the Maps plugin
-nav_order: 60
+title: Using maps 
+grand_parent: Building data visualizations
+parent: Using coordinate and region maps
+nav_order: 5
 redirect_from:
   - /dashboards/maps/
 ---
 
-# Using the Maps plugin
+# Using maps 
 
 With OpenSearch Dashboards, you can create maps to visualize your geographical data. OpenSearch lets you construct map visualizations with multiple layers, combining data across different indexes. You can build each layer from a different index pattern. Additionally, you can configure maps to show specific data at different zoom levels. OpenSearch maps are powered by the OpenSearch maps service, which uses vector tiles to render maps. 
 
-## Getting started
+## Creating a new map
 
-To get started, perform the following steps:
+You can create a new map from the **Maps** or **Visualize** workflows by performing the following steps: 
 
-1. On the top menu bar, go to **OpenSearch Plugins > Maps**.
-1. Select the **Create map** button. You can now see the default OpenSearch basemap.
-1. To examine the **Default map** layer configuration, in the **Layers** panel on the upper left of the map, select **Default map**, as shown in the following image.
+- To create a new map from the **Maps** workflow, perform the following steps:
+
+  1. On the top menu bar, go to **OpenSearch Plugins > Maps**. 
+  1. Select the **Create map** button. 
+
+- To create a new map from the **Visualize** workflow, perform the following steps:
+
+  1. On the top menu bar, go to **OpenSearch Dashboards > Visualize**.
+  1. Select the **Create visualization** button.
+  1. In the **New Visualization** dialog, select **Maps**.
```

Comment:
Global: Use "choose" to describe moving to a UI component such as a tab or pane or taking action on a button or menu.